src/App/component/Login.jsx

```diff
@@ -29,24 +29,37 @@ export default function Login() {
   const [username, setUsername] = useState("")
   const [password, setPassword] = useState("")
 
-
-  const login = () => {
+  const login = async () => {
     if (username === "" || password === "") {
       alert("不準啦馬的>///<")
     } else {
       let payload = {
         username: username,
         password: password
       }
-      Axios.post(`http://localhost:9100/pvs-api/auth/login`, payload)
-        .then((response) => {
-          localStorage.setItem("jwtToken", response.data)
-          goToSelect()
-        })
-        .catch((e) => {
-          alert(e.response.status)
-          console.error(e)
-        })
+      localStorage.setItem("jwtToken", await getJWToken(payload))
+      localStorage.setItem("memberId", await getMemberId())
+      goToSelect()
+    }
+  }
+
+  const getJWToken = async (payload) => {
```

Comment:
Suggest rename to `getJWTFrom`, and also rename `payload` to some other names.
```suggestion
  const getJWTFrom = async (credential) => {
```